bin/razor

```diff
@@ -50,7 +38,21 @@ https://tickets.puppetlabs.com/browse/RAZOR
 end
 
 begin
-  document = parse.navigate.get_document
+  navigate = parse.navigate
```

Comment:
It might be better to just have all the function declarations at the top, and then the main logic in a single begin block w/ all the rescues.

I kind of liked having the show_version and show_help stuff come before navigate.

Reply:
I think we should rip out setting the API url in the CLI::Parse class completely and just move it over to the Navigate class. The only thing that the CLI::Parse class should store is the user-provided API url from the command line, if any. I think it would be OK if the navigate class stores the API url as its own class variable (for now).

Because as it currently stands, the Parse class is setting the API url to the default if the user doesn't provide one, but the nav class also has code for doing that. That is a bit confusing, especially when the bulk of the API url setting code, such as the specific cases, are in the navigate class.